ecosystem/node-checker/src/evaluators/build_evaluators.rs

```diff
@@ -97,6 +98,11 @@ pub fn build_evaluators(
         evaluator_identifiers.iter().cloned().collect();
     let mut evaluators: Vec<EvaluatorType> = vec![];
 
+    BuildVersionEvaluator::add_from_evaluator_args(
```

Comment:
nit: if typing is a problem, we could consider adding a macro for this. So that we'd be able to do something like:
`build_evaluator!(BuildVersionEvaluator)` that could expand to what you have here.

e.g., here's an example of a `macro_rules!` declaration: https://github.com/aptos-labs/aptos-core/blob/ffb726685d8ea007a488d711cb2c73b910e0a11b/state-sync/state-sync-v2/data-streaming-service/src/stream_engine.rs#L29-L28

Reply:
I knew the day would come where it's better to write a macro than not. I'll take it for a spin.

Reply:
Okay it doesn't actually provide that much of a benefit: https://stackoverflow.com/questions/53716044/why-can-i-not-access-a-variable-declared-in-a-macro-unless-i-pass-in-the-name-of. I end up having this:
```
    build_evaluator!(
        BuildVersionEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        ConsensusProposalsEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        ConsensusRoundEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        ConsensusTimeoutsEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        LatencyEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        NetworkMinimumPeersEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        NetworkPeersWithinToleranceEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        StateSyncVersionEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        TpsEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
    build_evaluator!(
        TransactionPresenceEvaluator,
        evaluators,
        evaluator_identifiers,
        evaluator_args
    );
```

Which actually takes more lines.